.rubocop.yml

```diff
@@ -527,6 +528,10 @@ RSpec/NestedGroups:
 Layout/IndentHeredoc:
   Enabled: False
 
+Rspec/IteratedExpectation:
```

Comment:
I don't think this is necessary now, `.rubocop.yml` is managed by modulesync, but current master doesn't have any rubocop warnings, so as long as this PR didn't change spec_helper_acceptance, shouldn't be necessary.

For some reason, I don't see this when I check out the PR locally.